### PR TITLE
Theme Cicada update for Sidebar

### PR DESCRIPTION
**Pull request by marjohn56**

### CODE REVIEW

File: misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -1,9538 +1,8250 @@
-@charset "UTF-8";
```
Comment (fichtner) [inline misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/main.scss]: same here, but file seems to have been clobbered before

**Comment by marjohn56** [comment]
OK, I will pull the upstream files and take a closer look.

**Comment by marjohn56** [comment]
René want's to hold this, and Tukan. So can you revert the Tukan commit? I assumed as he had sent them to me they were ready.

**Comment by fichtner** [comment]
Sure, will roll back the tukan update.

**Pull request closed without merging**